docs/api/rest_api/rest_api_reference/input/ibexa-ai.raml

```diff
@@ -21,35 +21,162 @@ displayName: AI Actions
         body:
             application/vnd.ibexa.api.ai.AltText+json:
                 type: GenerateAltTextWrapper
-                example: !include examples/ai/POST/GenerateAltText.json.example
+                example: !include examples/ai/actions/execute/POST/GenerateAltText.json.example
             application/vnd.ibexa.api.ai.AltText+xml:
                 type: GenerateAltTextWrapper
-                example: !include examples/ai/POST/GenerateAltText.xml.example
+                example: !include examples/ai/actions/execute/POST/GenerateAltText.xml.example
             application/vnd.ibexa.api.ai.RefineText+json:
                 type: RefineTextInputWrapper
-                example: !include examples/ai/POST/RefineTextInput.json.example
+                example: !include examples/ai/actions/execute/POST/RefineTextInput.json.example
             application/vnd.ibexa.api.ai.RefineText+xml:
                 type: RefineTextInputWrapper
-                example: !include examples/ai/POST/RefineTextInput.xml.example
+                example: !include examples/ai/actions/execute/POST/RefineTextInput.xml.example
         responses:
             200:
                 description: Action executed successfuly
                 body:
                     application/vnd.ibexa.api.ai.AltText+json:
                         type: AltTextWrapper
-                        example: !include examples/ai/POST/AltText.json.example
+                        example: !include examples/ai/actions/execute/POST/AltText.json.example
                     application/vnd.ibexa.api.ai.AltText+xml:
                         type: AltTextWrapper
-                        example: !include examples/ai/POST/AltText.xml.example
+                        example: !include examples/ai/actions/execute/POST/AltText.xml.example
                     application/vnd.ibexa.api.ai.RefineText+json:
                         type: RefineTextWrapper
-                        example: !include examples/ai/POST/RefineText.json.example
+                        example: !include examples/ai/actions/execute/POST/RefineText.json.example
                     application/vnd.ibexa.api.ai.RefineText+xml:
                         type: RefineTextWrapper
-                        example: !include examples/ai/POST/RefineText.xml.example
+                        example: !include examples/ai/actions/execute/POST/RefineText.xml.example
             400:
                 description: Error - the request is invalid
             401:
                 description: Error - the user is not authorized to execute an AI action.
             404:
                 description: Error - the given Action Configuration identifier does not exist
+
+/actions:
+    get:
+        displayName: List action configurations
+        description: Returns a list of action configurations
+        queryParameters:
+            limit:
+                description: The number of results per page
```

Comment:
Seems like we are inconsistent in putting fullstops at the end of query paramater descriptions, but the ones with fullstop largely prevail. 
I'd add fullstops then, even though these are not full sentences. Alternatively, we'd want to go through all descriptions and amend them to have fullstops in sentences and no fullstops in nominal sentence fragments